### DISCUSSION TITLE
How to add dialogues?

### DISCUSSION THREAD

**Discussion by KolzeYT** [YoshiCrafter29/YoshiCrafterEngine#174] [Q&A]
I need help i want to add dialogues and i dunno how.

**Comment by IMdaAG** [comment]
Write your dialogue in a .txt file with :dad: representing the opponent, :bf: bf and :gf: gf.
Then copy the weeb-dialogue.hx from Friday Night Funkin'/cutscenes and then modify whatever the frick you want (songs, dialogue boxes, portraits, music during dialogue)
